mimic/rest/identity_api.py

```diff
@@ -669,6 +688,521 @@ def list_tenants(self, request):
                 'message': ("No valid token provided. Please use the 'X-Auth-Token'"
                             " header with a valid token.")}})
 
+    @app.route('/v2.0/services', methods=['GET'])
+    def list_external_api_services(self, request):
+        """
+        List the available external services that endpoint templates
+        may be added to.
+
+        .. note:: Part of the OS-KSADM extension.
```

Comment:
I've added URLs where I can still grab them...but it seems that OS-KSCATALOG is no longer being listed by OpenStack's documentation :frowning: so I used the URL I already had in there. It's the best I can do but also a reflection of the sad state of affairs regarding URLs and how easily they become non-existent.